pkg/api/http/http.go

```diff
@@ -433,6 +434,17 @@ func (a *api) onOutputBindingMessage(w nethttp.ResponseWriter, r *nethttp.Reques
 		}
 	}
 
+	if baggageHeaders := r.Header.Values(diagConsts.BaggageHeader); len(baggageHeaders) > 0 {
+		baggageString := strings.Join(baggageHeaders, ",")
+
+		if _, err3 := otelBaggage.Parse(baggageString); err3 == nil {
+			if req.Metadata == nil {
+				req.Metadata = map[string]string{}
+			}
+			req.Metadata[diagConsts.BaggageHeader] = baggageString
+		}
```

Comment:
Again, we shouldn't be ignoring the errors here. This is a malformed request.

Reply:
Fair. It was optional metadata, not core business logic is why I was simply not propagating it along if parsing failed. I'll update it to error out entirely